<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-7I9F7TC" name="用户中心模块测试" tests="4" failures="3" timestamp="18 Nov 2020 09:17:03 GMT" time="0.149" errors="0">
  <testcase name="addUser" time="0.01" classname="com.qa.test.OrganizeTest"/>
  <testcase name="queryUser" time="0.012" classname="com.qa.test.OrganizeTest">
    <failure type="java.lang.AssertionError" message="expected [5] but found [70]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [70]
	at com.qa.test.OrganizeTest.queryUser(OrganizeTest.java:77)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- queryUser -->
  <testcase name="editUser" time="0.063" classname="com.qa.test.OrganizeTest">
    <failure type="java.lang.AssertionError" message="expected [5] but found [70]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [70]
	at com.qa.test.OrganizeTest.queryUser(OrganizeTest.java:77)
	at com.qa.test.OrganizeTest.editUser(OrganizeTest.java:88)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- editUser -->
  <testcase name="delteUser" time="0.045" classname="com.qa.test.OrganizeTest">
    <failure type="java.lang.AssertionError" message="expected [5] but found [70]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [70]
	at com.qa.test.OrganizeTest.queryUser(OrganizeTest.java:77)
	at com.qa.test.OrganizeTest.delteUser(OrganizeTest.java:102)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- delteUser -->
</testsuite> <!-- 用户中心模块测试 -->
